packages/material-ui/src/useMediaQuery/useMediaQuery.js

```diff
@@ -1,5 +1,13 @@
 import * as React from 'react';
 import { getThemeProps, useTheme } from '@material-ui/styles';
+import useEnhancedEffect from '../utils/useEnhancedEffect';
+
+// useEnhacedEffect does nothing on server side, so when
+// the unit test is executed it appears a warning
+// To pass the test we add a validation where
+// useEnhacedEffect is used in any environment but 'test'
```

Comment:
@mnajdova @oliviertassinari Hey guys! I added these few lines, let me know if that is enough. Also, I just noticed that a one check failed, I'm not sure why it happened, but I'll review the changes again!

Reply:
I’d say it would be better if we can update the test like this https://github.com/mui-org/material-ui/blob/268fff9d7536532307ccb8dd685dc97256f9bd08/packages/material-ui/src/NoSsr/NoSsr.test.js#L19 rather than doing this condition. Regarding the failing argos check, we can ignore (will approve in the end if it fails again).

Reply:
okay! i'll do it! Thanks!

Reply:
Perfect, thank you!